### PR TITLE
MGMT-4049 Add id information to the inventory disk - The id field identify the disk to the UI instead of computing it from the by-path and by-id field  

### PR DESCRIPTION
**Pull request by michaellevy101**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -4270,6 +4270,8 @@ definitions:
   disk:
     type: object
     properties:
+      id:
```
Comment (avishayt) [inline swagger.yaml:4273]: Neither the commit message nor the JIRA describes what this new field is or why it's needed.

Comment (omertuc) [inline swagger.yaml:4273]: Please add a swagger `description:` to:
`by_path`
`by_id`
`id`

so we understand what exactly is the difference between all of them

Comment (michaellevy101) [reply swagger.yaml:4273]: done

Comment (michaellevy101) [reply swagger.yaml:4273]: Added the info to the design document, PR and commit description https://docs.google.com/document/d/1L1_y3NDvx_NTNKK2VZrc9IFabnrd37P_jpiyS9gjAnw/edit

File: swagger.yaml [hunk 2]
```diff
@@ -4270,6 +4270,9 @@ definitions:
...
+        type: string
+        description: Determine the disk's unique identifier which is the by-id field is exist and fallback to the by-path field
```
Comment (omertuc) [inline swagger.yaml:4275]: nit: typo "is exist" -> "if it exists"

**Comment by YuviGold** [comment]
Your PR is failing since you didn't run `generate-all` before commiting.

**Review by osherdp — APPROVED** [review]
/lgtm
/approve

**Pull request merged**